### PR TITLE
Usmanafzal/website 577 update readme

### PR DESCRIPTION
**Pull request by UsmanAfzal**
### [WEBSITE-466] - Updates to README

- Changed description of repository
- Used make command rather than npm

### CODE REVIEW

File: src/javascripts/map.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+				} else {
+					console.log(request.statusText);
+				}
+			}
+		};
+		request.send();
+	}
+
+	ukp_getJsonFile('../json/mapData.json', function (data) {
+		// console.log(data);
```
Comment (jakemulley) [inline src/javascripts/map.js]: Rogue console.log

File: src/javascripts/map.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		var mymap = L.map('mapbox').setView([51.505, -0.09], 10);
+		L.tileLayer(
+			'https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token=pk.eyJ1IjoibWFwYm94IiwiYSI6ImNpejY4NXVycTA2emYycXBndHRqcmZ3N3gifQ.rJcFIG214AriISLbB6B5aw', {
```
Comment (jakemulley) [inline src/javascripts/map.js]: Where are these values ({id}, {z}, {x}, {y}) stored?

File: src/json/mapData.json [hunk 3]
```diff
@@ -0,0 +1,2427 @@
+{
```
Comment (jakemulley) [inline src/json/mapData.json]: This is _way_ too big. Should this even be in the repository, or should we be getting it from a data store?

**Comment by jakemulley** [comment]
My thoughts: since the map stuff included here is a prototype that won't make it to production, these additions should be made into a users repository rather than into a repository owned by UKParliament.

File: README.md [hunk 4]
```diff
@@ -11,23 +11,23 @@ Static HTML prototypes created by the [Parliamentary Digital Service][pds] for P
...
-After setting up, run `make serve`. Browse to http://localhost:3000 to view the prototype.
+After setting up, run `make serve`. Browse to http://localhost:3000 to view Pugin (UK Parliament Brand) on example prototypes.
 
 ## Contributing
 If you wish to submit a bug fix or feature, you can create a pull request and it will be merged pending a code review.
 
 1. Fork the repository
 1. Create your feature branch (`git checkout -b my-new-feature`)
-1. Commit your changes (`git commit -am 'Add some feature'`)
+1. Commit your changes (`git commit -m 'Add some feature'`)
```
Comment (jakemulley) [inline README.md]: This misses out the need to add files into a commit.

**Comment by UsmanAfzal** [comment]
The base branch used was incorrect, commit closed

**Pull request closed without merging**